src/towncrier/test/test_format.py

```diff
@@ -462,3 +462,100 @@ def test_line_wrapping_disabled(self):
             versiondata={"name": "MyProject", "version": "1.0", "date": "never"},
         )
         self.assertEqual(output, expected_output)
+
+    def test_trailing_block(self):
+        """
+        Make sure a newline gets inserted before appending the issue number, if the
+        newsfragment ends with an indented block.
+        """
+
+        fragments = {
+            "": {
+                (
+                    "1",
+                    "feature",
+                    0,
+                ): "text::\n\n    def foo(): ..."
+            }
+        }
+        expected_output = """MyProject 1.0 (never)
+=====================
+
+Features
+--------
+
+- text::
+
+      def foo(): ...
+
+  (#1)
+
+
+"""
+
+        # TODO: I copy-pasted the below lines from previous test, they probably contain
```

Comment:
this needs to be cleanup before merge.

Reply:
I .. don't think it contains irrelevant crap currently? But I'm not overly familiar with your test structure so it's very possible it still does.